### PR TITLE
[Issue 84] Add table of explainers

### PR DESCRIPTION
**Pull request by joannasendorek**
## Description

Please include a summary of the changes.

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

No tests required.

## Checklist:

- [ ] I have updated all related `pyproject.toml` files.
- [ ] I have updated lock files.
- [x] I have set an Assignee and Reviewers.
- [x] I have labeled the PR correctly.
- [x] I have performed a self-review of my own code.
- [ ] I have covered my code and changes with unit tests.
- [ ] I have annotated types as extensively as possible.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] All code references and dependencies will work.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -233,3 +233,32 @@ be introduced. In the end, the module should work with all types of tasks (NLP,
...
+| CV_INPUT_X_GRADIENT_EXPLAINER           | yes                                  |
+| CV_LAYER_INPUT_X_GRADIENT_EXPLAINER     | yes                                  |
+| CV_DEEPLIFT_EXPLAINER                   | yes, if non-random baseline provided |
+| CV_LAYER_DEEPLIFT_EXPLAINER             | yes, if non-random baseline provided |
+| CV_DEEPLIFT_SHAP_EXPLAINER              | yes, if non-random baseline provided |
+| CV_LAYER_DEEPLIFT_SHAP_EXPLAINER        | yes, if non-random baseline provided |
+| CV_DECONVOLUTION_EXPLAINER              | yes                                  |
+| CV_LAYER_CONDUCTANCE_EXPLAINER          | yes                                  |
+| CV_SALIENCY_EXPLAINER                   | yes                                  |
+| CV_GUIDED_BACKPOPAGATION_EXPLAINER      | yes                                  |
```
Comment (adamwawrzynski) [inline README.md:264]: Add [XRAI](https://arxiv.org/abs/1906.02825) algorithm to the table.

**Review by adamwawrzynski — APPROVED** [review]
Looks good.

File: README.md (hunk 1 above)
Comment (joannasendorek) [reply README.md:264]: Added :)

**Review by kamilrzechowski — APPROVED** [review]
Thanx, that's exactly what I had in my mind.

**Pull request merged**